scorep/__main__.py

```diff
@@ -11,6 +12,39 @@ def _err_exit(msg):
     sys.exit(1)
 
 
+def print_help():
+    print("""
+Usage: python -m scorep [options] [--] your_program.py [args]
+
+Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed:
+
+  --help                   Show this help message and exit.
+  --mpi                    Enable MPI instrumentation (equivalent to --mpp=mpi).
```

Comment:
is this already marked as deprecated?

Reply:
the code does not support your claim. if so, `--mpp=mpi` should be handled by Python too, but its not, and `--mpi` just passes `--mpp=mpi` to `scorep-config`. The `README` uses `--mpp=mpi` and notes that "The traditional `--mpi` does still work". so advertising it in the `--help` does not make sense to me. `--mpp=mpi` should, and thats why I suggested to add a "Common options to Score-P"-section, which should also mention `--thread=pthread` or `--thread=ompt`, if thats supported by Score-P